prisma/schema.prisma

```diff
@@ -226,6 +226,9 @@ model Event {
   // data stored in the event
   data Json?
 
+  // metadata stored in the event
+  additionalMetadata Json?
```

Comment:
can we not name this `metadata`? Would be a little cleaner. Or is this conflicting with existing metadata names?

Reply:
I'm going to leave this as-is, we can certainly change the column name later. I imagine this is preventing `metadata` from being an overloaded field in the API. I think a more apt name would be `userMetadata`?